pythonmarketo/client.py

```diff
@@ -236,7 +237,43 @@ def create_lead(self, lookupField, lookupValue, values):
             ]
         }
         return self.post(data)
+    
+    def run_request_campaign(self,campaignID,leadsID,values):
+        def add_curly(k):
+           return '{{'+k+'}}' 
+        append_array={add_curly(k):v for k, v in values.items()}
+        leads_list=[]
+        token_dict={}
+        token_list=[]
+        leads_dict={}
+        for k,v in append_array.items():
+            token_dict['name']=k
+            token_dict['value']=v
```

Comment:
token_dict is a variable reqired only in the for loop why initialise it outside?  is the variable necessary, it can be simply  token_list = [{'name':k, 'value':v} for k, v in append_array.items()] or simply you don't even need append_array it can be 
token_list = [{'name':'{{' + k + '}}', 'value':v} for k, v in values.items()]